.github/CODEOWNERS

```diff
@@ -1,2 +1,38 @@
-# wasmCloud team members
-* @rvolosatovs @autodidaddict @brooksmtownsend @thomastaylor312 @connorsmith256
+# Default to org maintainers
+* @wasmCloud/org-maintainers
+
+# .github CI
+.github @wasmCloud/ci-maintainers
+
+# nix
+.github/actions/build-nix @rvolosatovs
+.github/actions/install-nix @rvolosatovs
+.github/workflows/nix.yml @rvolosatovs
+flake.* @rvolosatovs
```

Comment:
Doesn't this mean that any change to flakes would require my review for merge?
Doesn't this mean that if we enforce code owner reviews and I'm on PTO, we won't ever be able to merge anything related to flakes? (e.g. a component adapter update)

Reply:
I think the main issue is most of the rest of us aren't comfortable enough with how nix works to be confident. I am pretty sure I can check the flakes, but I also don't know that I'd spot issues